.docker/remove-volumes.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+source .docker/.volumes
+
+for volume in "${volumes[@]}"; do
+    if docker volume ls --filter name="${volume}" | grep "${volume}"; then
```

Comment:
Can we add `docker volume ls -q` to remove headings? And perhaps a comment to indicate that `grep` is used to provide an exit code, making the condition truthy or falsey?